docs/overview/set-up-your-dev-environment.md

```diff
@@ -70,17 +70,25 @@ If you created your project manually, install and configure the linter with the
    npm install eslint-plugin-office-addins --save-dev
    ```
 
-1. In the root of the project, create a text file named **.eslintrc.json**, if there isn't one already there. Be sure it has properties named `plugins` and `extends`, both of type array. The `plugins` array should include `"office-addins"` and the `extends` array should include `"plugin:office-addins/recommended"`. The following is a simple example. Your **.eslintrc.json** file may have additional properties and additional members of the two arrays.
-
-   ```json
-   {
-     "plugins": [
-       "office-addins"
-     ],
-     "extends": [
-       "plugin:office-addins/recommended"
-     ]
-   }
+1. In the root of the project, create a text file named **eslint.config.js** (or **.mjs**), if there isn't one already there. Be sure to inherit the recommended configuration for `eslint-plugin-office-addins`. The `plugins` array should include `eslint-plugin-office-addins`. The following is a simple example that includes settings for TypeScript. Your **eslint.config.js** file may have additional properties and configurations.
+
```

Comment:
Technically `plugins` is not an array anymore . . . it is an object.

Reply:
Good catch! Corrected.